infrastructure/modules/sqs/variable.tf

```
@@ -70,6 +70,11 @@ variable "dlq_visibility_timeout" {
   type    = number
   default = 0
 }
+variable "dlq_message_retention" {
```

Sorry for being picky!
```suggestion
}

variable "dlq_message_retention" {
```